### DISCUSSION TITLE
Blur is broken

### DISCUSSION THREAD

**Discussion by Necora1** [end-4/dots-hyprland#897] [Q&A]
When I turn on blur in the right panel, everything turns to be just text and at the top of the screen configs errors are shown: Config error in file /home/gash/.config/hypr/hyprland/colors.conf at line 9: rgba() expects Iength of 8 characters (4 bytes) or 4 comma separates values ; Config error in file /home/gash/.config/hypr/hyprland.conf at line 9: Config error in file /home/gash/.config/hypr/hyprland/colors.conf at line 9: rgba() expects length of 8 characters (4 bytes) or 4 comma separated values

**Comment by zcroll** [comment]
same problem i had just reinstall hyperland and remove all the config related and cache 
freech install it  going to work
